central/processindicator/datastore/datastore_impl.go

```diff
@@ -19,6 +19,10 @@ import (
 	pkgSearch "github.com/stackrox/rox/pkg/search"
 )
 
+const (
+	deleteBatchSize = 5000
```

Comment:
For my understanding: How many indicators are in a table depending on the cluster size?